keras/engine/sequential.py

```diff
@@ -163,11 +163,6 @@ def add(self, layer):
                     # to the input layer we just created.
                     layer(x)
                     set_inputs = True
-                else:
```

Comment:
Can we guarantee that these blocks can't be touched for all use-cases?

Reply:
These blocks can be touched, but the variables created inside are not used.

Reply:
LGTM